### ISSUE TITLE
Access violation when trying to print with POS printer

### ISSUE THREAD

**Issue by JanRajnoha** [microsoft/WindowsAppSDK#1222]
**Describe the bug**
I don't know, which repo is the best for report, but here are the most active people, so sorry if it is unrelated.

I have POS printer Rongta RP80 with support DLL POSDLL.dll. Printing on WPF is absolutely okay, but with WinUI app it is problem. Printer is connected via USB, drivers are installed and windows test print is working.
![image](https://user-images.githubusercontent.com/16069721/129110544-2e516cd2-bf9e-40cb-b3f7-09aff42b0d46.png)
Is any solution for this problem? Any hack or workaround to solve it? Or is possible to solve it with unpackaged app instead of packaged?

Printer is okay, is connected and opened, but commands fail.

**Steps to reproduce the bug**

1. WinUI app with library POSDLL.dll and trying to print

**Expected behavior**
Should print

**Version Info**
ProjectReunion - 0.8.1

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (22000.120)              | Yes  |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Comment by StephenLPeters** [comment]
I think WindowsAppSDK is more approriate for this.

**Comment by andrewleader** [comment]
Thanks for reporting this! @AdamBraden would you be the right person to initially take "Things that worked in unpackaged apps but don't work in packaged apps" like this issue?

**Comment by AdamBraden** [comment]
@JanRajnoha - the attached project appears to be winforms app, not using the WindowsAppSDK nor WinUI. Can you provide a github repo of a WinUI project?

Thanks,
Adam

**Comment by JanRajnoha** [comment]
I create repo with solution of WPF and WinApp SDK, where I have problem. Currently I have only solution in project for customer.

**Comment by JanRajnoha** [comment]
Maybe, repo won't be necessary. I found out, when I yesterday was trying to make new Epson pos printer functional, I installed many and many different drivers and so on and during creating test repo for you @AdamBraden, I was shocked, because Rongta printer worked. I think, one thing, what wasn't mentioned, was OPOS for .NET, which I installed during Epson installation. So, for this time, it is working, and I will try to confirm this solution.
